micro-core/src/main/java/com/aol/micro/server/utility/HashMapBuilder.java

```diff
@@ -25,6 +28,15 @@ public Builder(K key, V value) {
 			build = new HashMap<K, V>();
 			build.put(key, value);
 		}
+		public Builder(Map<K,V> map) {
+			build = new HashMap<K, V>();
```

Comment:
Yeah - I think 

      build = new HashMap<K, V>(map);

is better in that regard and cleaner.

It's probably not a copy, as the builder is initialized with the map, but calling code can add more values. We could add an immutableBuild method that did this though (and returned an immutable map or view on the map).

At the moment, it's a convenience class to make configuration simpler, and isn't used in anything performance critical.